redux-fapp/src/reducers/reducer.card.js

```diff
@@ -0,0 +1,35 @@
+const reducerCard = (
+  state = {
+    loading: false,
+    userData: null,
+    error: null,
+    status: null
+  },
+  action
+) => {
+  switch (action.type) {
+    case "CARD_REQUESTING":
```

Comment:
import this from action 
example:- import *as cardAction from "../action/card.action.js


case cardAction.CARD_REQUESTING: